front/src/applications/operationalStudies/components/MacroEditor/consts.ts

```diff
@@ -159,3 +159,113 @@ export const DEFAULT_PACED_TRAIN_PAYLOAD: Pick<
 };
 
 export const DEFAULT_TIME_WINDOW = new Duration({ hours: 2 });
+
+const TRAIN_CATEGORY_TO_NGE: {
+  trainCategory: TrainCategory;
+  colorRef: string;
+}[] = [
+  { trainCategory: 'HIGH_SPEED_TRAIN', colorRef: 'HST' },
+  { trainCategory: 'INTERCITY_TRAIN', colorRef: 'IC' },
+  { trainCategory: 'REGIONAL_TRAIN', colorRef: 'RE' },
+  { trainCategory: 'COMMUTER_TRAIN', colorRef: 'S' },
+  { trainCategory: 'FREIGHT_TRAIN', colorRef: 'F' },
+  { trainCategory: 'FAST_FREIGHT_TRAIN', colorRef: 'FF' },
+  { trainCategory: 'TRAM_TRAIN', colorRef: 'TT' },
+  { trainCategory: 'WORK_TRAIN', colorRef: 'W' },
+  { trainCategory: 'NIGHT_TRAIN', colorRef: 'NT' },
+  { trainCategory: 'TOURISTIC_TRAIN', colorRef: 'TO' },
+];
+
+export const OSRD_TRAINRUN_CATEGORY_MAPPING: Record<TrainCategory, TrainrunCategory> =
+  TRAIN_CATEGORY_TO_NGE.reduce(
+    (acc, { trainCategory, colorRef }, index) => {
+      acc[trainCategory] = {
+        id: index,
```

Comment:
```suggestion
        id: index + 1,
```
I think we try to avoid `id` that could be equal to `0`. It can break checks in condition like 
```ts
if (train.id) {
  // ...
}

Reply:
Done